var/spack/repos/builtin/packages/portage/package.py

```diff
@@ -17,14 +17,22 @@ class Portage(CMakePackage):
 
     # tarballs don't have submodules, so use git tags
     version('develop', branch='master', submodules=True)
+    version('1.2.2', tag='v1.2.2', submodules=True)
     version('1.1.1', tag='v1.1.1', submodules=True)
     version('1.1.0', tag='v1.1.0', submodules=True)
 
+    # fabs() needs math.h for gcc-7, got fixed in version above 1.2.2
+    patch('gcc-7.patch', when='@1.2.2 %gcc@7:')
```

Comment:
Please change this to `when='@:1.2.2 %gcc@7:'` in order to match the comment.  I'm not worried that you haven't personally tested past versions.

Reply:
I would prefer not touching and possibly breaking an old version by with unapplying patch, I will just update the comment.